definitions/scenarios/satellite_upgrade.rb

```diff
@@ -48,6 +48,7 @@ def compose
         Checks::ForemanTasks::Invalid::CheckPendingState, # if foreman-tasks
         Checks::ForemanTasks::Invalid::CheckPlanningState, # if foreman-tasks
         Checks::ForemanTasks::NotRunning, # if foreman-tasks
+        Checks::Pulpcore::NoRunningTasks, # if pulpcore
```

Comment:
Seems valid for update as well to check?

Reply:
I don't follow what you mean here

Reply:
Should the `update` scenario also include this?